en-GB.lproj/Main.strings

```diff
@@ -1,42 +1,42 @@
 
 /* Class = "UILabel"; text = "Export TFP0"; ObjectID = "0lM-oV-NGn"; */
-"0lM-oV-NGn.text" = "Export TFP0";
+"0lM-oV-NGn.text" = "Exporteren TFP0";
 
 /* Class = "UIButton"; normalTitle = "Clean Diagnostics Data"; ObjectID = "2He-sb-oZG"; */
-"2He-sb-oZG.normalTitle" = "Clean Diagnostics Data";
+"2He-sb-oZG.normalTitle" = "Schone diagnosegegevens";
```

Comment:
"Diagnosegegevens opschonen" would be a more appropriate sentence seeing as it is an action you can perform.